### DISCUSSION TITLE
Intersecting Freehand Type for Selection Tool

### DISCUSSION THREAD

**Discussion by melieconiek** [opentoonz/opentoonz#6574] [Ideas]
Looking for freehand selection type that selects the objects inside the lasso as well as the objects it intersects with. In most cases this is a faster and more precise selection mode. :)

An example below: selecting a set of objects and then deselecting another that was inside the selection--and a mock-up of a simple intersecting freehand selection. In nearly all cases, the stroke you have to draw for the selection is smaller and allows you to more easily avoid objects you don't wish to select.

_(In this simplified example you'd probably just select the lines individually, but for more complex drawings this isn't as easy.)_

![a](https://user-images.githubusercontent.com/42298081/85635646-636bcb80-b64c-11ea-97ec-d5e0f9520039.gif) ![b](https://user-images.githubusercontent.com/42298081/85635654-65358f00-b64c-11ea-9433-9870eacc4b03.gif)

![image](https://user-images.githubusercontent.com/42298081/85636083-85198280-b64d-11ea-864d-ec34d22860b3.png)

**Comment by ghost** [comment]
I remember having a similar request, it would be a good marriage to be able to switch with the current behavior which also has a lot of advantages.
